handlers/bin/command-not-found

```diff
@@ -27,13 +28,17 @@ def print_found(rows):
               file=sys.stderr)
     print(file=sys.stderr)
     print(_('Try installing with:\n   '), end=' ', file=sys.stderr)
+    installcmd = INSTALL_CMD
     if os.getuid() != 0:
-        print(SUDO_CMD, end=' ', file=sys.stderr)
-    print(INSTALL_CMD, end=' ', file=sys.stderr)
+        installcmd = SUDO_CMD + installcmd
     if len(set([i[1] for i in rows])) > 1:
-        print(_('<selected_package>'), file=sys.stderr)
+        print(installcmd + _('<selected_package>'), file=sys.stderr)
```

Comment:
LGTM, but maybe you want to rethink the order as 'Try installing with:' is showed even in the case that later is installed.

If you do not agree feel free to merge

Reply:
I dont like the added complexity of that... or do you see a simple way to do it?